### DISCUSSION TITLE
The diff message is almost invisible

### DISCUSSION THREAD

**Discussion by btwear** [NeogitOrg/neogit#962] [Q&A] [answered]
Hi,

As the following image, the line of the diff message is almost invisible.
I have changed several themes but seems has no much difference.
Is there some configuration I can tune for this?
Thanks for the help 🙏

![image](https://github.com/NeogitOrg/neogit/assets/17814132/0947b2ec-5183-45cb-a653-19405c2fb75b)

**Comment by CKolkey** [comment, marked as answer]
Check out `:h neogit_highlights`. From the look of it, it's the `NeogitDiffDelete` highlight group. You can just define it to be something more to your liking, or open a PR/Issue on the theme's repo.

**Reply by btwear** [reply]
It works! thanks a lots🚀
